CHANGES.md

```diff
@@ -74,6 +74,8 @@ To be released.
     not related with [Kademlia protocol][Kademlia].  [[#594], [#627]]
  -  `Swarm<T>` became not to check least recently used peer every time when
     new peer is fetched.  [[#627]]
+ -  When there is no block state when evaluating an action, it is changed to
+    calculate the state from the beginning.  [[#645]]
```

Comment:
How about this?

> `IAction` became guaranteed that the given `IActionContext.PreviousStates.GetState()` never throws `IncompleteBlockStatesException`.  Instead, now it may calculate the incomplete states from the beginning if necessary.